pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter.go

```diff
@@ -82,7 +82,7 @@ func admitHotplug(newVolumes, oldVolumes []v1.Volume, newDisks, oldDisks []v1.Di
 		return webhookutils.ToAdmissionResponse([]metav1.StatusCause{
 			{
 				Type:    metav1.CauseTypeFieldValueInvalid,
-				Message: "number of disks does not equal the number of volumes",
+				Message: fmt.Sprintf("number of disks (%d) does not equal the number of volumes (%d)", len(newDisks), len(newVolumes)),
```

Comment:
I'm just curious, is this change directly related to this PR?

Reply:
Actually it isn't. Sorry, next time I'll point it out in a comment here.
It's just that I changed this for debugging then thought it's better to stay that way. Opening a PR especially for that seemed a bit extreme to me.